packages/domain-search/src/components/search-results/item.tsx

```diff
@@ -15,45 +10,15 @@ interface SearchResultsItemProps {
 
 export const SearchResultsItem = ( { domainName }: SearchResultsItemProps ) => {
 	const [ domain, ...tlds ] = domainName.split( '.' );
-
-	const suggestionBadges = useDomainSuggestionBadges( domainName );
-
-	const suggestion = useSuggestion( domainName );
-
 	const tld = tlds.join( '.' );
 
-	const notice = useMemo( () => {
-		if ( suggestion.hsts_required ) {
-			return createInterpolateElement(
-				sprintf(
-					/* translators: %s is the TLD of the domain */
-					__(
-						'All domains ending in <strong>%(tld)s</strong> require an SSL certificate to host a website. When you host this domain at WordPress.com an SSL certificate is included. <a>Learn more</a>.'
-					),
-					{
-						tld: `.${ tld }`,
-					}
-				),
-				{
-					a: <a href={ localizeUrl( HTTPS_SSL ) } target="_blank" rel="noopener noreferrer" />,
-					strong: <strong />,
-				}
-			);
-		}
-
-		if ( suggestion.dot_gay_notice_required ) {
-			return __(
-				'Any anti-LGBTQ content is prohibited and can result in registration termination. The registry will donate 20% of all registration revenue to LGBTQ non-profit organizations.'
```

Comment:
Was this snippet moved somewhere else? I think we're required to show this by the registry

Reply:
This notice should be served by the backend. Now that you mention it, I wasn't able to list any .gay domain in the new domain search. Here's a screenshot comparing the old and the new UI:

<img width="2992" height="1180" alt="image" src="https://github.com/user-attachments/assets/96a2473d-302e-4c35-98ed-8718dfef3838" />

Are you seeing the same issue? cc @zaguiini

Reply:
I think that's because of the `vendor`. Can you check which one do you see in the suggestion fetching request? You can see it in the query parameters within the dev tools.